GEOSens_GridComp/GEOS_EnsGridComp.F90

```diff
@@ -1604,6 +1604,24 @@ subroutine SetServices(gc, rc)
                                            RC=STATUS  )
   VERIFY_(STATUS)
 
+  call MAPL_AddExportSpec(GC                         ,&
+    LONG_NAME          = 'CN_autotrophic_respiration' ,&
```

Comment:
I suggest we call this "CN_total_autotrophic_respiration".  Would also need to be changed in https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/1129